add-graphite-scalers-doc (#425)
kedacore/keda-docs: content/docs/2.3/scalers/graphite.md

+++
title = "Graphite"
layout = "scaler"
availability = "v2.3+"
maintainer = "Community"
description = "Scale applications based on metrics in Graphite."
go_file = "graphite_scaler"
+++

### Trigger Specification

This specification describes the `graphite` trigger that scales based on metrics in Graphite.

```yaml
triggers:
- type: graphite
  metadata:
    # Required
    serverAddress: http://<graphite-host>:81
    metricName: request-count
    query: stats.counters.http.hello-world.request.count.count # Note: query must return a vector/scalar single element response
    threshold: '100'
    queryTime: '-10minutes'
```

> **Parameter list:**

Reviewer: Would be nice to extend the description of the parameters

- `serverAddress` - Address of Graphite

> - `metricName` - Metric name to use

Reviewer: The `metricName` is needed for Graphite or is this for KEDA purposes?

- `query` - Query to run
- `threshold` - Value to start scaling for

> - `queryTime` - Query Time to from 

Reviewer: What are the possible values for queryTime? What is the syntax?

+++
title = "Graphite"
layout = "scaler"

> availability = "v1.0+"

Reviewer 2: ```suggestion
availability = "v2.3+"
```

maintainer = "Community"

> description = "Scale applications based on Graphite."

Reviewer 2: ```suggestion
description = "Scale applications based on metrics in Graphite."
```

go_file = "graphite_scaler"
+++

### Trigger Specification

> This specification describes the `graphite` trigger that scales based on a Graphite.

Reviewer 2: ```suggestion
This specification describes the `graphite` trigger that scales based on metrics in Graphite.
```

```yaml
triggers:
- type: graphite
  metadata:

>     # Required

Reviewer 2: I presume all of them are required?

    serverAddress: http://<graphite-host>:81
    metricName: request-count
    query: stats.counters.http.hello-world.request.count.count # Note: query must return a vector/scalar single element response
    threshold: '100'

>     queryTime: '-10minutes'

Reviewer 2: Is this a standard statsd unit or something we have defined?

Author: 是定义的示例

```

**Parameter list:**

- `serverAddress` - Address of Graphite
- `metricName` - Metric name to use
- `query` - Query to run
- `threshold` - Value to start scaling for
- `queryTime` - Query Time to from 

> 

Reviewer 2: Nit
```suggestion
```
